merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -113,7 +114,8 @@ public List<String> validateActivityArguments(final String missionModelId, final
   {
     try {
       // TODO: [AERIE-1516] Teardown the missionModel after use to release any system resources (e.g. threads).
-      return this.loadConfiguredMissionModel(missionModelId).validateActivity(activity);
+      return this.loadUnconfiguredMissionModel(missionModelId)
+                 .validateActivityArguments(activity);
```

Comment:
`MissionModelFacade#validateActivityInstantiations` is no longer used anywhere except in tests. Can we remove it, and update the tests to use the unconfigured variant you've added here?

Reply:
Tests have been updated, and method has been removed :+1: